pygpu/__init__.py

```diff
@@ -14,10 +14,8 @@ def get_include():
 
 from .version import fullversion as __version__
 
-from .tests import main
-if hasattr(main, "NoseTester"):
-    test = main.NoseTester().test
-else:
-    def test():
-        raise ImportError("The nose module is not installed."
-                          " It is needed for pygpu tests.")
+
+def test():
+    from .tests import main
+    if hasattr(main, "NoseTester"):
+        main.NoseTester().test()
```

Comment:
It will raise the same error as before.  The previous error message was never encountered since it would just fail to import nose all the time.